src/services/recim/series.ts

```diff
@@ -99,7 +99,7 @@ const putSeriesSchedule = async (schedule: UploadSeriesSchedule): Promise<Series
   http.put(`recim/series/schedule`, schedule);
 
 const scheduleSeriesMatches = async (seriesID: number): Promise<Match[]> =>
-  http.post(`recim/series/${seriesID}/autoschedule`);
+  http.post(`recim/series/${seriesID}/autoschedule`, {});
```

Comment:
Do we need a body tag for this?

Reply:
yep, the other PR adds a comment in there stating `temporary solution`. There is a new parameter to handle Round Robin Match Cap & Number of Ladder Matches